apps/pano/app/utils/index.ts

```diff
@@ -93,14 +93,24 @@ export function validatePassword(password: unknown): password is string {
 
 export function getExternalPostURL(post: PostWithCommentCount) {
   const location = global.location;
-  const postUrl = `${location.origin}/posts/${post.slug}/${post.id}`;
+  const postUrl = `${location.origin + getPostSlugLink(post)}`;
```

Comment:
if you are not gonna interpolate, please remove the backticks:

```suggestion
  const postUrl = location.origin + getPostSlugLink(post);
```